tests/backup/backup_test.go

```diff
@@ -5708,6 +5711,80 @@ var _ = Describe("{ViewOnlyFullBackupRestoreIncrementalBackup}", func() {
 
 })
 
+var _ = Describe("{DeleteUsersRole}", func() {
+	var testrailID = 58089
```

Comment:
No need to give this. Directly pass the ID in `StartTorpedoTest`

Reply:
declared since passed 2 places